### ISSUE TITLE
Provided CP (Windows) not working with Cached Credentials

### ISSUE THREAD

**Issue by Garoun** [multiOTP/multiotp#20]
_The Issues option isn't available in your multiOTP/multiOTPCredentialProvider repo so posting it here, seeing as though the CP is provided as part of the download._

**Problem:**
CP works fine as long as AD is available, unplugging the network cable to simulate an offline AD results in 'incorrect pin' error from the CP and blocks login.  However, you'll notice the multiotp server says the login was approved and valid.

**Setup/Troubleshooting:**
- Configured the server/client using AD Sync (confirmed working)
- Configured the client machine to test Cache, TOTP, and Scratch login (confirmed working)
- Renamed test client setup
- Ran Provider installer, held at 'Test' phase
- Ensured Provider installed version of multiotp matched my test client setup (confirmed working)
- Completed install of Provider
- Tested TOTP, and Scratch logins after Reboot and Lock with and without initial user db created (confirmed working)
- Tested Cached credentials: 
-- Ensured db files were still available
-- Locked machine (also tested a fresh reboot that had the same result)
-- Unplugged network cable
-- Attempted to unlock machine (**Failed with error: Incorrect Pin**)
-- The included log below is from multiotp, I'm not sure how to view/create logs for the CP

**Multiotp log output with network cable unplugged using multiOTPCredentialProvider:**
```
multiotp 5.0.4.8
Your script is running from C:\multiOTP\
2017-07-27 10:53:18	debug		Debug	Debug: *parameter(s) received: [REDACTED_USER] [REDACTED_TOTP]
2017-07-27 10:53:21	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:23	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:26	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:28	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:28	notice	[REDACTED_USER]	User	OK: User [REDACTED_USER] successfully logged in with TOTP token
2017-07-27 10:53:28	debug		Debug	Debug: *0 OK: Token accepted
2017-07-27 10:53:28	debug		Debug	Debug: *Attributes sent to the RADIUS server: Filter-Id += "Domain Users",Filter-Id += "multiotp-debug-group"
========================================
multiotp 5.0.4.8
Your script is running from C:\multiOTP\
2017-07-27 10:53:36	debug		Debug	Debug: *parameter(s) received: [REDACTED_USER] [REDACTED_SCRATCH]
2017-07-27 10:53:38	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:41	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:43	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:46	debug		Client-Server	Warning: Host [REDACTED_AD] on port 8112,http: not reached before a timeout of 3 seconds.
2017-07-27 10:53:46	notice	[REDACTED_USER]	User	Ok: User [REDACTED_USER] successfully logged in with a scratch password
2017-07-27 10:53:46	debug		Debug	Debug: *0 OK: Token accepted
2017-07-27 10:53:46	debug		Debug	Debug: *Attributes sent to the RADIUS server: Filter-Id += "Domain Users",Filter-Id += "multiotp-debug-group"
```

**Comment by Garoun** [comment]
Appears to have been caused by a configuration issue on my part.

When doing the initial install, accidentally using "," instead of ";" when entering the servers will cause the cache check to fail in the CP even though multiOTP validates this token as ok.

**Comment by multiOTP** [comment]
The "," instead of ";" has been fixed in the forthcoming version 5.0.5.x
Official separator is still ";", but [space] and "," will be allowed too.
Thanks for your feedback
Regards

**Comment by Garoun** [comment]
Glad I could help!
